utils/storage/storage_openbsd.go

```diff
@@ -0,0 +1,18 @@
+// Public Domain
```

Comment:
I'm a bit confused as to why this isn't failing the license check

Reply:
jk - it is failing

Reply:
Thank you.  `scripts/lint.sh` didn't run right away on adJ/OpenBSD because it requires GNU grep, however I installed  it with `doas pkg_add ggrep`  and changed a little the script (see https://github.com/ava-labs/avalanchego/commit/c91cddbd4532537152da4661ed6abbd6d8a437dc --I could improve it more to propose another PR).

Regarding the license header:
1. I expected that Ava Labs would do that change.  But well ...
2. Since that file was created in 2024, I tried just with  the year 2024 in `// Copyright (C) 2024, Ava Labs, Inc. All rights reserved. `  but it didn't pass te test, so I just used `// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.`